content/pt_PT/faq.md

```diff
@@ -1,34 +1,31 @@
 
-## How can I contribute?
+## Como posso contribuir?
 
-Everyone can help. io.js adheres to a [code of conduct](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), and contributions, releases, and contributorship are under an [open governance](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme) model.
+Todos podem ajudar. O io.js adere a um [código de conduta](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), e as contribuições, lançamentos e autoria estão sob um modelo de [governança aberta](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme).
 
-To get started, there are open [ discussions on GitHub](https://github.com/iojs/io.js/issues), and we'd love to hear your feedback.
-Becoming involved in discussions is a good way to get a feel of where you can help out further. If there is
-something there you feel you can tackle, please [make a pull request](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-contributions).
+Para começar, existem [ discussões abertas no GitHub](https://github.com/iojs/io.js/issues), e adoraríamos ouvir o seu feedback. Envolver-se nas discussões é uma boa maneira de saber onde poderá ajudar mais. Se existir algo que sente que pode fazer, por favor [crie um pull request](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-contributions).
 
-In addition, using [Nodebug.me](http://nodebug.me/) is a good way to help Triage the issues in the backlog.
+Além disso, usar o [Nodebug.me](http://nodebug.me/) é uma boa maneira de ajudar a fazer a triagem dos issues no backlog.
 
-## Where do discussions take place?
+## Onde é que as discussões acontecem?
 
-There is an #io.js channel on Freenode IRC. We keep logs of the channel [here](http://logs.libuv.org/io.js/latest).
+Existe um canal #io.js no IRC Freenode. Guardamos os logs do canal [aqui](http://logs.libuv.org/io.js/latest).
 
-## What is open source governance?
+## O que é governança de software livre?
 
-Open source governance advocates the application of the philosophies of the open source and open content movements in order to enable any interested party to add to the creation of the end product, as with a wiki document. Legislation is democratically opened to the general citizenry, employing their collective wisdom to benefit the decision-making process and improve democracy. [[source]](https://en.wikipedia.org/wiki/Open-source_governance)
\ No newline at end of file
+A governança de software livre defende a aplicação das filosofias dos movimentos software livre e conteúdo aberto de modo a permitirem a todos os interessados que possam contribuir na criação do produto final, tal como acontece com um documento wiki. A legislação é democraticamente aberta à cidadania em geral, aplicando a sua inteligência coletiva para o benefício do processo de tomada de decisão e melhoria da democracia. [[fonte]](https://en.wikipedia.org/wiki/Open-source_governance)
```

Comment:
What do you say about using "de modo a permitir que todos os interessados possam contribuir"?